media-sound/ocp: Version bump and general update of recipe

mywave82 opened the pull request:
This depends on new version of app-arch/ancient #7793

Begasus commented:
Needs https://github.com/haikuports/haikuports/pull/7793 to be merged before, so added wip for now.

Begasus commented on media-sound/ocp/ocp-0.2.102.recipe:
```diff
@@ -0,0 +1,99 @@
+INSTALL()
+{
+	make install
+	addAppDeskbarSymlink $binDir/ocp OpenCubicPlayer
```
This should be $commandBinDir :)
```diff
-	addAppDeskbarSymlink $binDir/ocp OpenCubicPlayer
+	addAppDeskbarSymlink $commandBinDir/ocp OpenCubicPlayer
```

mywave82 replied:
This is now fixed

Begasus replied:
Thanks!

mywave82 commented:
> Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.

Fixed

> PS: minor nit-pick/suggestion... PR title's "by tradition" are formatted like: `<package_name>: <what you did with the recipe>`, in this case, I would use something like: `ocp: update version to 0.2.102`

Fixed

mywave82 commented:
If there any more comments, do not be alarmed that I am not able to fix this before 5th of February due to a work trip :-D

Begasus commented:
This one is fine by me, only needs libancient to be merged before we can merge this. :)

mywave82 commented:
> > > Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.
> > 
> > 
> > Fixed
> 
> I'm still seeing only one commit adding the new recipe (instead of the suggested "git mv / modify / add / commit" cycle. Not that important, I guess, just mentioning it because maybe the "fix" got lost among other changes? git can be a PITA :-D

I will check again when I'm back home next week. I did git reset master, move out old new recipe, git mv, and overwrite recipe with new content before git commit. Strange strange :)

mywave82 commented:
> > > > Maybe the version bump could be made by `git mv ocp-0.2.2.recipe ocp-0.2.102.recipe`, doing the necessary modifications to ocp-0.2.102.recipe, and `git add ocp-0.2.102.recipe` / `git commit` that.
> > > 
> > > 
> > > Fixed
> > 
> > 
> > I'm still seeing only one commit adding the new recipe (instead of the suggested "git mv / modify / add / commit" cycle. Not that important, I guess, just mentioning it because maybe the "fix" got lost among other changes? git can be a PITA :-D 

I think I got it correct this time

mywave82 commented:
> Needs #7793 to be merged before, so added wip for now.

It has now been merged

Begasus commented:
Thanks!
